test/distributed/fsdp/test_composable_fsdp.py

```diff
@@ -209,6 +213,57 @@ def _param_init_fn(module: nn.Module):
             self.assertEqual(composable_param, fsdp_wrapped_param)
 
 
+class TestFSDPRuntime(FSDPTest):
+    """Tests composable FSDP runtime."""
+
+    @property
+    def world_size(self) -> int:
+        return 2
+
+    @skip_if_lt_x_gpu(2)
+    def test_training(self):
+        """Tests training (forward, backward, optimizer)."""
+        device = torch.device("cuda")
+        local_model = Model(device=device)
```

Comment:
do we have support for composable FSDP + meta device? Is there a source of truth where we can find the feature set covered by composable?

Reply:
> do we have support for composable FSDP + meta device?

I believe there should be because the composable FSDP constructor includes the same module materialization logic as the normal FSDP constructor.

> Is there a source of truth where we can find the feature set covered by composable?

This is difficult to document right now since we are still prototyping. As I continue testing and thinking about the design, I may realize some sharp edges that prevent feature parity. I will try to stabilize soon. (same for `use_orig_params=True`)